micropython/certificates/README.md

```diff
@@ -0,0 +1,90 @@
+
+## MicroPython Certifi
+
+This is an approach to include Root CA certificates from the most common Certificate Authorities following 
+
+current [ssl certificates usage statistics](https://w3techs.com/technologies/overview/ssl_certificate)
+
+### Ca-bundle Instructions
+---------------------
+#### 1. Download ca-bundle.crt
+`ca-bundle.crt` is obtained using *curl's* script `mk-ca-bundle.pl` from [curl](https://github.com/cur)
```

Comment:
The url here looks truncated?

Reply:
yes, my bad 😓 thanks for pointing this out 👍🏼